projects/1.18-fabric/assets/bosses-of-mass-destruction/bosses_of_mass_destruction/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+{
+    "entity.bosses_of_mass_destruction.comet": "坠星",
+    "entity.bosses_of_mass_destruction.lich": "夜歌",
+    "entity.bosses_of_mass_destruction.blue_fireball": "幽焰弹",
+    "entity.bosses_of_mass_destruction.soul_star": "魂星",
+    "entity.bosses_of_mass_destruction.obsidilith" : "龙晶结构体",
+    "entity.bosses_of_mass_destruction.gauntlet" : "下界巨拳",
+    "entity.bosses_of_mass_destruction.void_blossom" : "虚空之花",
+    "entity.bosses_of_mass_destruction.spore_ball" : "孢子",
+    "entity.bosses_of_mass_destruction.petal_blade" : "落英",
+  
+    "block.bosses_of_mass_destruction.obsidilith_end_frame" : "黑曜石祭坛",
+    "block.bosses_of_mass_destruction.obsidilith_rune" : "黑曜石符文",
+    "block.bosses_of_mass_destruction.sealed_blackstone" : "密封的黑石",
+    "block.bosses_of_mass_destruction.gauntlet_blackstone" : "錾制远古黑石",
+    "block.bosses_of_mass_destruction.chiseled_stone_altar" : "錾制石头祭坛",
+    "block.bosses_of_mass_destruction.void_blossom_block" : "虚空之花刷怪笼",
+    "block.bosses_of_mass_destruction.void_lily" : "空无百合",
+    "block.bosses_of_mass_destruction.void_lily.tooltip" : "它的花粉将指引你至何方？",
+    "block.bosses_of_mass_destruction.void_blossom" : "虚空之花",
+    "block.bosses_of_mass_destruction.vine_wall" : "灌木屏障",
+  
+    "itemGroup.bosses_of_mass_destruction.items" : "Bosses of Mass Destruction",
+  
+    "item.bosses_of_mass_destruction.soul_star" : "灵魂之星",
+    "item.bosses_of_mass_destruction.soul_star.tooltip" : "它将带你至何方？",
+    "item.bosses_of_mass_destruction.ancient_anima" : "古代灵魂",
+    "item.bosses_of_mass_destruction.blazing_eye" : "烈焰之眼",
+    "item.bosses_of_mass_destruction.obsidian_heart" : "黑曜石之心",
+    "item.bosses_of_mass_destruction.void_thorn" : "虚空荆棘",
+    "item.bosses_of_mass_destruction.crystal_fruit" : "水晶果实",
+    "item.bosses_of_mass_destruction.crystal_fruit.tooltip" : "获得短暂的再生和抗性提升效果",

```

```suggestion
    "item.bosses_of_mass_destruction.crystal_fruit.tooltip" : "获得短暂的生命恢复和抗性提升效果",
```
详见[译名标准化](https://minecraft.fandom.com/zh/wiki/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96)